app/templates/index.hbs

```diff
@@ -1,5 +1,5 @@
 
-{{#fbf-nav type="default" primaryButton="drawer" as |section|}}
+{{#fbf-nav type="default" title="SAPP Store" logo=false primaryButton="drawer" as |section|}}
```

Comment:
I think so 2! but I also think that is something we can look at when we change the general design, like the new colors (orange blue) and such